migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/differ_database.rs

```diff
@@ -56,23 +56,13 @@ impl<'a> DifferDatabase<'a> {
 
         // First insert all namespaces from the previous schema.
         for namespace in schemas.previous.describer_schema.namespace_walkers() {
-            let namespace_name = if flavour.lower_cases_table_names() {
```

Comment:
I checked and MySQL lowercasing settings also affect database names.

Reply:
aah ok, I thought that hadn't been addressed. I'll drop that commit then.

Reply:
pushed